packages/ui/src/theme/css/component/field.scss

```diff
@@ -11,12 +11,8 @@
     font-size: var(--amplify-components-field-large-font-size);
     gap: var(--amplify-components-field-large-gap);
   }
+}
 
-  // Required to override docs styling
-  // css-s90uia p { margin-bottom: 1rem; }
-  // @TODO: remove styling from docs
-  .amplify-field__description,
```

Comment:
Fixed this.

Reply:
![fVtcfEXWQJQUbsF1sH](https://user-images.githubusercontent.com/6165315/141932369-85632a9e-5c3a-4e59-b239-e1b4d8e27971.gif)